README.md

```diff
@@ -316,7 +325,7 @@ fig.savefig("docs/example_multiple_lines.png")
 
 ![Recall with multiple lines](docs/example_multiple_lines.png)
 
-## Metrics
+## Statistical results
```

Comment:
I'm struggling with the wording here. The API uses the word "metrics". I decided to use metrics internally because we are talking about metrics and scores, not statistics. This keeps the API and wording in line with the wording used machine learning packages like sklearn and tensorflow (we depend on both packages). 

However, the CLI uses "stats", which is legacy. I was thinking about renaming this. This is actually the best moment for renaming. I don't see any stats we want to include here in the nearby future, as dataset statistics are moved to datatools extension.

Reply:
Yes, I agree. But be aware that metrics are the wss, erf and recall. Their time series results are presented in a plot (via the plotting command) or as point estimates (via the stats command). Actually, stats should be used for computing statistics, like the mean wss or the variance of erf across mutiple simulation runs. 

I propose to change stats to metrics in the CLI (we can add stats in a later stage if we add functionality  for computing means/variance of metrics) and, in the readme, refer to the point estimates as numerical results, output or something similar.